### PR TITLE
add Types

### PR DESCRIPTION
**Pull request by sbh1114564087**

### CODE REVIEW

File: lib/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,79 @@
+import { ReactNode } from 'react';
+import { Animated } from 'react-native';
+
+export interface IToastOpts {
+	data: string;
+	duration?: number;
+	position?: number;
```
Comment (taoqf) [inline lib/index.d.ts]: export enum positionType{ TOP = 1, CENTER= 2, BOTTOM = 3 }
declare var position: positionType;
export enum durationType{  SHORT = 1, LONG = 2 }

**Comment by mochixuan** [comment]
positon、heigh is error

**Comment by taoqf** [comment]
pls point out what's wrong with `height` so I can fix it

**Comment by mochixuan** [comment]
Already modified in version 1.1.3

**Comment by mochixuan** [comment]
> pls point out what's wrong with `height` so I can fix it

Already modified in version 1.1.3

**Comment by sbh1114564087** [comment]
Now，I am sorry to tell you that I don't what's wrong about it ,maybe know tomorrow------------------ 原始邮件 ------------------
发件人: "mochixuan"<notifications@github.com>
发送时间: 2019年7月22日(星期一) 晚上7:08
收件人: "mochixuan/react-native-smart-tip"<react-native-smart-tip@noreply.github.com>;
抄送: "sbh1114564087"<1114564087@qq.com>;"Author"<author@noreply.github.com>;
主题: Re: [mochixuan/react-native-smart-tip] add Types (#4)


 
pls point out what's wrong with height so I can fix it
  
Already modified in version 1.1.3
 
&mdash;

**Pull request merged**